packages/amplify-category-analytics/src/provider-utils/awscloudformation/cloudformation-templates/pinpoint-cloudformation-template.json

```diff
@@ -121,13 +119,9 @@
             {
               "Effect": "Allow",
               "Principal": {
-                "Service": [
-                  "lambda.amazonaws.com"
-                ]
+                "Service": ["lambda.amazonaws.com"]
```

Comment:
There seem to be a lot of unrelated formatting changes. Do you have a linter that is turned on? Is there any chance that you can separate the linter changes from the upgrade changes? This makes the pull request more difficult to review with all the additional changes.

Reply:
@danielleadams Thanks for the suggestion. I have turned off the linter. I have committed and pushed only changes that update the nodejs version.

Reply:
great - thank you.